javascript-loops-practice/filter-out-nulls.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutNulls */
+
+function filterOutNulls(values) {
+  const filteredArray = [];
+  for (let i = 0; i < values.length; i++) {
+    if (Boolean(values[i]) === true) {
```

Comment:
Works, but we can also just check for `null` directly. `if (values[i] !== null)`